test-suite/test-squid-conf.sh

```diff
@@ -15,8 +15,33 @@ top_builddir=$1
 sbindir=$2
```

Comment:
FYI: Here is where we stand with regard to ERROR and WARNING messages produced by Squid when parsing squid-conf-tests configuration files.

Messages already covered by the official master/v6 code:

```
ERROR: configuration failure: POSIX regcomp(3) failure
```

Messages covered by this PR:

```
ERROR: action 'menu' (line 26) already has a password

./squidconf/time_units.conf(56) url_rewrite_timeout: WARNING: missing time unit, using deprecated default 'second'
```

The messages below are not yet covered. Please let me know if you want me to cover any of these in this PR. IMO, they all should to be addressed by changing how Squid works, which goes outside this particular PR scope. FWIW, Factory is already working on a PR related to the last two.

```
ERROR: Unrecognised basic auth scheme parameter '/dev/null'

WARNING: external_acl_type format %>{...} is deprecated. Use %>ha{Header}
WARNING: external_acl_type format %>{...} is deprecated. Use %>ha{Hdr:member}
WARNING: external_acl_type format %>{...} is deprecated. Use %>ha{Hdr:;member}
WARNING: external_acl_type format %>{...} is deprecated. Use %>ha{Hdr:Xmember}
WARNING: external_acl_type format %<{...} is deprecated. Use %<h{Header}
WARNING: external_acl_type format %<{...} is deprecated. Use %<h{Hdr:member}
WARNING: external_acl_type format %<{...} is deprecated. Use %<h{Hdr:;member}
WARNING: external_acl_type format %<{...} is deprecated. Use %<h{Hdr:Xmember}

ERROR: Can not open file empty.conf for reading
WARNING: empty ACL: acl Mark dstdomain "empty.conf"
```